projects/protobuf-cplusplus/project.yaml

```diff
@@ -0,0 +1,20 @@
+homepage: "https://developers.google.com/protocol-buffers"
+language: c++
+primary_contact: "p.antoine@catenacyber.fr"
+auto_ccs:
+- protobuf-oss-fuzz@google.com
```

Comment:
Can you add the same `auto_ccs`, and 'labels` as in https://github.com/google/oss-fuzz/blob/master/projects/protobuf-java/project.yaml ? 

@zhangskz FYI. Is this correct?

Reply:
Pushed :-)

Reply:
Thanks for updating the ccs to include Protobuf team! This looks correct for what would be needed for us to access / triage these bugs, but see my comment below.